include/gz/common/Console.hh

```diff
@@ -46,8 +46,7 @@ namespace gz
       /// initialized or false otherwise.
```

Comment:
We should remove the doxygen comment for `fileInitialize`.

Reply:
[a1488d5](https://github.com/gazebosim/gz-common/pull/627/commits/a1488d589d0a4d3b0d374f6d59e93cecbc6e3575)